demos/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "homepage": "https://github.hubspot.com/calling-extensions-sdk",
```

Comment:
The default GitHub Pages repository URL `<organization>.github.io/` would typically be exposed to the public - created CALL-3336 to follow up with a more secure hosting alternative.